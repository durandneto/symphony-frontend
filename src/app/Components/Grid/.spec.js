import React from 'react'
import renderer from 'react-test-renderer'

import Container from '.'

describe('Container', () => {
    it('Rendering Container without crashing', () => {
        const tree = renderer.create(<Container>Text 2</Container>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})