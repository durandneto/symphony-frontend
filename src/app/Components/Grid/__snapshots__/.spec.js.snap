// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Rendering Container without crashing 1`] = `
<div
  className="sc-AxjAm cbwcBb"
>
  Text 2
</div>
`;
